change/@fluentui-web-components-518b3512-c7f4-4073-91e2-880c782b0864.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentui-web-components-v3 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30819&commitId=8a82a7878e6ccbcbffbdfed4db8f79622f7ea5af&env=prod&clientType=FLUENTUI)


  <details><summary><b>Slider</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Slider.Slider Steps Vertical.normal.chromium.png|1| Changed|

</details>


  <details><summary><b>Text</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Text.Block.normal.chromium.png|995| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>

Reply:
Slider seems like noise, the other...very odd. Seems 1:1:
<img width="684" alt="image" src="https://github.com/microsoft/fluentui/assets/13071055/8bae39f8-bc43-4a5c-ad13-c559406203f8">

Reply:
@spmonahan thoughts on this? It's a revert with no other changes to styles...the slider one is noise. The second I have no idea what is going on.

Reply:
Looks like the candidate snapshot adds a background

![msedge_hp0fLYUeRg](https://github.com/microsoft/fluentui/assets/93940821/c7846847-4897-400e-a113-4f0576d7a099)

Reply:
Weird. Thanks for the vid. I'll look. I don't see it immediately, but I also need to get OOF for a bit :)

I'll get this resolved. Can we mark the slider as noise or accept it?

Reply:
@spmonahan looks like this may have *not* run with the prior change. The story has an inline style for background to _show_ this exactly. With the prior change the inline style would have been missing the `var()` meaning this would not have been applied. I think this is by design based on the `text.stories.ts` file including `background` with the value of colorBackgroundNeutral6.

Reply:
Agree that this is a legit change and fine to accept. I would have expected the previous change to bring this issue up but I didn't see `Text` flagged in that PR. The changes need to be accepted in the VRT tool.